Sources/ComposableArchitecture/Documentation.docc/Articles/MigrationGuides/MigratingTo1.11.md

```diff
@@ -62,7 +62,7 @@ To fix this deprecation you can use the new ``Shared/withLock(_:)`` method on th
 case .delayedIncrementButtonTapped:
   return .run { _ in
     @Shared(.count) var count
-    $count.withLock { $0 += 1 }
+    await $count.withLock { $0 += 1 }
```

Comment:
Looks that this should be part of MigratingTo1.12.md, not MigratingTo1.11.md

Reply:
Hi @larryonoff, yeah that's true. Want to PR a fix? If you don't have time we will get to it soon.